bmorph/evaluation/StatCor_Vis.py

```diff
@@ -0,0 +1,566 @@
+import numpy as np
+import xarray as xr
+import pandas as pd
+import seaborn as sns
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import sys
+import bmorph
+
+sns.set_context('talk')
+mpl.style.use('seaborn-bright')
+mpl.rcParams['figure.figsize'] = (12, 9)
```

Comment:
These lines can probably be removed, as they are configuration that a user should set themselves. If a user were to set them before importing this module then their preferences would be overwritten.